repo-agent/review-ui/review-api/pkg/api/handlers_dev.go

```diff
@@ -111,13 +76,14 @@ func (s *Server) fetchAndPopulateDevSandboxes(ctx context.Context, namespace, re
 			// The UI card key is `sandbox.name`. If we use branch name, it must be unique per repo.
 			// Let's use the DevSandbox name as the key in Redis to match deletion logic.
 
-			key := fmt.Sprintf("dev:ns:%s:repo:%s:dev:%s", namespace, repo, item.GetName())
-			if err := s.Redis.HSet(ctx, key,
-				"sandbox", item.GetName(),
-				"branch", branch,
-				"branchURL", branchURL,
-				"sandboxReplica", fmt.Sprintf("%d", replicas),
-			).Err(); err != nil {
+			sandbox := models.DevSandbox{
+				Name:           item.GetName(),
+				Sandbox:        item.GetName(),
+				Branch:         branch,
+				BranchURL:      branchURL,
+				SandboxReplica: fmt.Sprintf("%d", replicas),
```

Comment:
Nit: we could probably stop casting everything to string if we're going to support real databases :-)